Sources/SwiftFormat/Utilities/FileIterator.swift

```diff
@@ -145,7 +145,7 @@ public struct FileIterator: Sequence, IteratorProtocol {
         // be displayed as relative paths. Otherwise, they will still be displayed as absolute
         // paths.
         let relativePath =
-          path.hasPrefix(workingDirectory.path)
+          path.hasPrefix(workingDirectory.path) && path.first != "/"
```

Comment:
Doesn’t this mean that we never relativize paths on Unix because all paths will start with a `/`? Don’t you want to check that `workingDirectory` is `/`? I might be misunderstanding your change though.

Reply:
Hmm... You seem to be correct here. As far as I can tell the reason for using relative paths is just for nice diagnostic output.

```
        // We attempt to relativize the URLs based on the current working directory, not the
        // directory being iterated over, so that they can be displayed better in diagnostics. Thus,
        // if the user passes paths that are relative to the current working directory, they will
        // be displayed as relative paths. Otherwise, they will still be displayed as absolute
        // paths.
```

And with `workingDirectory` being a constant of `.` this may take a bit more work. If we did change `workingDirectory` to be passed in then it would also allow for easier testing. I'll work on it a bit more and see what I can come up with.